### PR TITLE
Import contacts 

### PR DESCRIPTION
**Pull request by Llerena97**

### CODE REVIEW

File: app/views/import/contact.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<h1>Import#contact</h1>
+<% unless @contacts.nil?%>
```
Comment (Jucerweb) [inline app/views/import/contact.html.erb]: indentation

File: test/controllers/import_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```
Comment (Jucerweb) [inline test/controllers/import_controller_test.rb]: If you deleted this folder, you should delete this one, try to not always  use 'rails generate controller'

File: app/views/import/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Import#index</h1>
```
Comment (Jucerweb) [inline app/views/import/index.html.erb]: Delete this one, if you use `rails generate controller` delete all the unnecessary files!

File: app/assets/stylesheets/import.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the import controller here.
```
Comment (Jucerweb) [inline app/assets/stylesheets/import.scss]: Deleted this one

File: app/assets/javascripts/import.coffee [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Jucerweb) [inline app/assets/javascripts/import.coffee]: Deleted this one

File: app/controllers/import_controller.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class ImportController < ApplicationController
+  def index
+  end
+
+  def contact
```
Comment (Jucerweb) [inline app/controllers/import_controller.rb]: Why did you create this action ? if you have `index`

File: app/controllers/import_controller.rb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+class ImportController < ApplicationController
+
+  def contact
```
Comment (Jucerweb) [inline app/controllers/import_controller.rb]: Why are you still using this action called `contact`? The right way to list records is `index`.

**Comment by Llerena97** [comment]
Solved dismaker!

**Pull request merged**